### PR TITLE
traffic-manager

### PR DESCRIPTION
**Pull request by burkeholland**

### CODE REVIEW

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/traffic-manager.md](https://github.com/burkeholland/azure-docs-sdk-node/blob/patch-17/docs-ref-services/traffic-manager.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C14%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707141459353054-82%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-services/traffic-manager.md [hunk 1]
```diff
@@ -13,26 +13,40 @@ ms.devlang: nodejs
...
+  client.profiles.listAllInResourceGroup(resourceGroupName).then(profiles => {
+    profiles.map(profile => {
+      console.log(`found profile : ${profile.name}`);
+    });
+  });
+});
+```
 
-##Samples
+## Samples
```
Comment (clarkio) [inline docs-ref-services/traffic-manager.md]: Three "###"

File: docs-ref-services/traffic-manager.md [hunk 2]
```diff
@@ -13,26 +13,40 @@ ms.devlang: nodejs
...
 
-## Install the module with npm
-
-Use npm to install the Azure Traffic Manager module for Node.js
-
-### Management
+## Management Package
 
+### Install npm module
 ```bash
```
Comment (johnpapa) [inline docs-ref-services/traffic-manager.md]: Add this ...

> Install the Azure traffic manager npm module

File: docs-ref-services/traffic-manager.md [hunk 3]
```diff
@@ -13,26 +13,40 @@ ms.devlang: nodejs
...
+### Example
+
+Authenticate to Azure, create a new client and list all Traffic Managers for a given resource group.
+
+```javascript
+const msRestAzure = require('ms-rest-azure');
+const trafficManager = require('azure-arm-trafficmanager');
+
+msRestAzure.interactiveLogin().then(credentials => {
+  const client = new trafficManager(credentials, 'subscription-id');
```
Comment (johnpapa) [inline docs-ref-services/traffic-manager.md]: move subscription id up to a const

```javascript

const subscriptionId = 'your-subscription-id`;

```

File: docs-ref-services/traffic-manager.md (hunk 1 above)
Comment (johnpapa) [inline docs-ref-services/traffic-manager.md]: 2 is correct.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/traffic-manager.md](https://github.com/burkeholland/azure-docs-sdk-node/blob/patch-17/docs-ref-services/traffic-manager.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180103006906-82%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/traffic-manager.md](https://github.com/burkeholland/azure-docs-sdk-node/blob/patch-17/docs-ref-services/traffic-manager.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180423579419-82%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by johnpapa** [comment]
LGTM 

#sign-off

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/traffic-manager.md](https://github.com/burkeholland/azure-docs-sdk-node/blob/patch-17/docs-ref-services/traffic-manager.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707181328235480-82%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**